# Generated by Django 5.1.1 on 2024-10-12 19:38

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('authentication', '0002_wwcuser_profile_pic'),
    ]

    operations = [
        migrations.AlterField(
            model_name='wwcuser',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 30 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=30, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator], verbose_name='username'),
        ),
    ]
